### ISSUE TITLE
[Request] Fetch Show / Movie genre for NFO

### ISSUE THREAD

**Issue by dm2912** [CastagnaIT/plugin.video.netflix#583]
## Feature request
I'm submitting a proposal to ...
  - [x] new feature
  - [ ] change an existing feature

### Describe your request

If possible add genre for movies and shows when fetching nfo info. Does not need fetching for episodes just movie.nfo and show.nfo

#### Describe the use case

Being able to look through your library by genre (especially if you have a huge library) is a pretty fundamental part of kodi. Currently the netflix addon does not fetch genre info when writing nfo's

#### Describe an alternative

At the moment im rewriting all my show.nfo and movie.nfo with tiny media manager, but its a lot of work just to bring genres over

The other alternative would be to let Kodi prioritise online information for library items but with netflix this causes its own problems because they often have a very different episode numbering system to online sources (money heist for example)

#### Screenshots

For this with big libraries not having netflix items show up in genre menus (without making custom nfo) is quite frustrating

![image](https://user-images.githubusercontent.com/25871666/79670639-77163400-81bc-11ea-9889-fd501d667d27.png)

but lack of genres also makes some skins return no information lines

![image](https://user-images.githubusercontent.com/25871666/79670606-37e7e300-81bc-11ea-9ed4-1df8c92415dc.png)

In short, genre information for most people and their kodi libraries is a really important piece of information.

**Comment by CastagnaIT** [comment]
the information used to build the NFO files comes from the metadata
metadata has little information

there is no genre information contained, so it cannot be added
